src/persistence/models/card.py

```diff
@@ -0,0 +1,9 @@
+from peewee import UUIDField, TextField, BigIntegerField
+
+from src.persistence.models.base_model import BaseModel
+from pgvector.peewee import VectorField
+
+class Card(BaseModel):
+    id: UUIDField()
```

Comment:
Can we use the autogenerated from PostGres instead?

Reply:
done